libs/search-widget/src/tiles/tile.utils.ts

```diff
@@ -0,0 +1,5 @@
+export const markRegex = new RegExp(/<\/*mark>/, 'g');
```

Comment:
I think NERs can have some attributes in their <mark> tags, like `mark class="ner" family="something">`.
So your regexp should capture more, like:
```/<\/*mark[^>]*>/```